easybuild/tools/options.py

```diff
@@ -568,6 +588,13 @@ def _postprocess_list_avail(self):
             self.log.info(msg)
         else:
             print msg
+
+        # cleanup tmpdir
+        try:
+            shutil.rmtree(self.options.tmpdir)
+        except OSError as err:
+            raise EasyBuildError("Failed to clean up temporary directory %s: %s", self.options.tmpdir, err)
```

Comment:
so don't catch the exception? it will bubble up to the script/gui calling your function here, and that script/gui can then inform the user about the problem, and quit what it's doing.
And maybe that script can handle certain errors, but not others, but now you're masking OSErrors and other errors alike to EasyBuildErrors, so it's hard for the calling script to distinguish between them, and to know what to do.